tasks/fdsys.py

```diff
@@ -239,7 +248,12 @@ def extract_sitemap_subject_from_url(url, how_we_got_here):
     # 113th Congress, "S." (senate) bills).
     m = re.match(re.escape(fdsys_baseurl) + r"bulkdata/(.*)/([^/]+)/sitemap.xml$", url)
     if m:
-        return { "bulkdata": True, "collection": m.group(1), "grouping": m.group(2) }
+        return_data = { "bulkdata": True, "collection": m.group(1), "grouping": m.group(2) }
+        congress_match = re.match(r"^([0-9]+)", m.group(2))
+        if congress_match:
```

Comment:
Let's say `if return_data["collection"] == "BILLSTATUS" and congress_match:` to make it explicit that this applies to the BILLSTATUS data.